e2e-tests/version-service/run

```diff
@@ -34,8 +34,8 @@ function apply_version {
 				{"op":"replace","path":"/spec/userLabels/pgo-version","value":"'${version_to_upgrade}'"}
 			]'
 	sleep 10
-
-	wait_generation_uniformity ${cluster} ${target_revision}
+	echo $target_revision
```

Comment:
Don't thing we need such a line here

Reply:
https://github.com/percona/percona-postgresql-operator/pull/183/commits/38c2a769adc31480d6aa072ff8a5ccb40e8722c5